pod/perlfunc.pod

```diff
@@ -3582,6 +3582,9 @@ X<goto> X<jump> X<jmp>
 
 =for Pod::Functions create spaghetti code
 
+C<goto> transfers execution to a specified point in the program. Unlike a
+subroutine call, it does not return.
```

Comment:
Am i being excessively pedantic when I think that goto &sub; does return, just not to where it was goto'ed from? I think this should be 'goto LABEL' only.

Reply:
@mauke, can you respond to @demerphq's comment above?  Thanks.

Reply:
`goto` is a control transfer. Execution flows into `goto`, but does not come out again. That's why I think it is correct to say "goto does not return".
(If you use `goto &sub`, the subroutine does return eventually, of course, but `goto` itself does not.)

Reply:
I've reworded it, hopefully making it less ambiguous.